webCommander/centralHub/views.py

```diff
@@ -51,3 +61,11 @@ def get_artists(request):
         "<option value='{val}'>{val}, Popularity: {pop}</option>".format(val=row[0], pop=row[2]) for row in response
     )
     return HttpResponse(options)
+
+def get_artist_choices(request):
```

Comment:
rewriting this I came up with mostly the same code as above, I think this view serves another purpose, let's integrate all later ...